src/contexts/providers.context.tsx

```diff
@@ -297,7 +283,10 @@ const ProvidersProvider: FC<PropsWithChildren> = (props) => {
           if (account) {
             void connectMetamaskProvider({ env, web3Provider, account });
           } else {
-            setConnectedProvider({ status: "failed", error: "No connected accounts found" });
+            setConnectedProvider({
+              status: "failed",
+              error: "",
```

Comment:
I think that you forgot the error here: `"No connected accounts found"`

Reply:
It is a conscious change, otherwise, if we cannot do the automatic login we will show this error and I don't think we want that. We just want to let the user log in again right?